### DISCUSSION TITLE
Roadmap to fix XSLX  vulnerabilty

### DISCUSSION THREAD

**Discussion by digital-codes** [opensource9ja/danfojs#609] [General]
The master branch has a dependency on a vulnerable XLSX version which is fixed in the dev branch already. What keeps you from upgrading the master branch for so long? Any help on particular topics required?

**Comment by ZandercraftGames** [comment]
Agreed. It's a year later and they still have these changes sitting on the dev branch. When will we get a new NPM release with these changes?

**Comment by RahulDas-dev** [comment]
They are not maintaining this repo.

If anyone is interested, we can clone it and create a new package with the same functionality.

**Reply by digital-codes** [reply]
Jeah, not much happened here in the last 2 years ... Well, browser based data processing in not so interesting given the limitations imposed by CORS when accessing resources. Anyway, browser based demos or educational tools are usefull, I believe. I started a couple of thing a while ago, basically aiming for a simplified version of https://orangedatamining.com/ or https://www.knime.com/ in the browser to show people how to use open data (and to get some data skills). 
Not sure if it makes sense to continue on this repo (which is a pity, though) ...

**Reply by ZandercraftGames** [reply]
Looks like we just got some updates! Woo!

**Reply by risenW** [reply]
Yeah, sorry for the delay on this. I'm reviewing existing PRs, and fixing some high priority issues. Will be releasing a new version tomorrow.

**Comment by ZandercraftGames** [comment]
Awesome! Thanks so much! :)

________________________________

**Comment by risenW** [comment]
New version 1.2.0 has been released with this fix.
